src/articles/ArticlePreview.js

```diff
@@ -105,10 +113,36 @@ export default function ArticleOverview({
         isArticleSaved={isArticleSaved}
         setIsArticleSaved={setIsArticleSaved}
       />
+
       <s.Title>{titleLink(article)}</s.Title>
-      <s.Difficulty>{difficulty}</s.Difficulty>
-      <s.WordCount>{article.metrics.word_count}</s.WordCount>
+      <s.ArticleContent>
+        {article.img_url && <img alt="" src={article.img_url} />}
+        <s.Summary>{article.summary}</s.Summary>
+        <div class="stats">
+          <s.Difficulty>{difficulty}</s.Difficulty>
+          <s.WordCount style={{ marginRight: "1em" }}>
+            {article.metrics.word_count}
+          </s.WordCount>
+        </div>
+      </s.ArticleContent>
 
+      <div>
+        {!dontShowImage && (
```

Comment:
now that we have an image, it would make sense to qualify this as being `source_icon` or `source_logo`?

Reply:
I will updated the name to be feed_icon_name

![image](https://github.com/zeeguu/web/assets/17390076/aefaf3f7-c774-4a5a-a649-642fa7e247d7)